pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -173,12 +173,12 @@ func (s *requested) ExecuteInbound(msg *stateMachineMsg, thid string, ctx *conte
 
 		err := json.Unmarshal(msg.payload, invitation)
 		if err != nil {
-			return nil, nil, nil, fmt.Errorf("unmarshalling failed: %s", err)
+			return nil, nil, nil, fmt.Errorf("unmarshalling failed: %w", err)
 		}
 
 		action, connRecord, err := ctx.handleInboundInvitation(invitation, thid, msg.connRecord)
 		if err != nil {
-			return nil, nil, nil, fmt.Errorf("handle inbound invitation failed: %s", err)
+			return nil, nil, nil, fmt.Errorf("handle inbound invitation failed: %w", err)
```

Comment:
We don't need to add `failed` to a wrapped error messages. 

Same comment for similar messages as well.

Reply:
done